source/documentation/get-all-messages.html.md.erb

```diff
@@ -11,13 +11,19 @@ Use the GET endpoint [Get all messages for an excise movement using the movement
 
 The consignor ID must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
 
+## Path parameters
+
+Parameters must be entered in lowercase.
+
+| Parameter | Type | Details |
+| -- | -- | -- |
+| lrn | String | Filter messages by local reference number (LRN) |
```

Comment:
The details of this should probably be something like "The local reference number of the movement"

Reply:
Thanks